config/sync/config_ignore.settings.yml

```diff
@@ -8,6 +8,7 @@ ignored_config_entities:
   - '~core.entity_view_mode.node.*'
   - ~core.menu.static_menu_link_overrides
   - ~dblog.settings
+  - '~dpl_library_agency.general_settings:reservation_sms_notifications_disabled'
```

Comment:
Consider whether we want to ignore all settings under `dpl_library_agency.general_settings`.

If they are ment to be configured by the individual library we probably do not want to control them from the configuration management system.